ALUM/ALUM/Views/LoginPageView.swift

```diff
@@ -0,0 +1,200 @@
+//
```

Comment:
Though already discussed during team meeting, wanted to reiterate on the MVVM architecture for this View. So lets make this LoginPageView satisfy MVVM. 

Here's what chat-gpt has to say about making this View satsify MVVM. 
>The Model-View-ViewModel (MVVM) design pattern is a widely used pattern in modern mobile development, and it can be applied to a login screen as well. The basic idea of MVVM is to separate the user interface and its logic from the underlying data model, making the code more organized and maintainable.

>Here's a step-by-step guide to designing an MVVM-based login screen:

>1. Model: Create a model that will store the user credentials (e.g. username and password). The model should be simple, containing only the required properties and logic.

>2. ViewModel: Create a ViewModel class that acts as a bridge between the Model and the View. The ViewModel should expose the data from the Model and provide the required logic for the login process, such as validation and communication with the server.

>3. View: Create the View for the login screen. The View should be responsible for presenting the data from the ViewModel and allowing the user to interact with it. The View should also bind to the properties and commands exposed by the ViewModel.

>4. Communication: Establish communication between the View and the ViewModel. The View should bind to the properties and commands exposed by the ViewModel, and the ViewModel should notify the View when the data changes.

>5. Validation: Add validation logic to the ViewModel to ensure that the user inputs are correct. The ViewModel should validate the user credentials before sending a request to the server.

>6. Server Communication (in our case firebase): Implement the logic for communicating with the server in the ViewModel. This can be done using a network library or by making HTTP requests directly.

>7. Error handling: Implement error handling in the ViewModel to handle errors that may occur during the login process, such as invalid credentials or network errors. The ViewModel should propagate the error to the View so that the user can be notified.

>By following these steps, you can create an MVVM-based login screen that is organized, maintainable, and easy to test.

These are just guidelines so adapt whichever seems reasonable to you for this case.